Userland/Libraries/LibJS/Runtime/NumberConstructor.cpp

```diff
@@ -46,7 +46,7 @@ ThrowCompletionOr<void> NumberConstructor::initialize(Realm& realm)
     define_direct_property(vm.names.parseFloat, realm.intrinsics().parse_float_function(), attr);
     define_direct_property(vm.names.EPSILON, Value(EPSILON_VALUE), 0);
     define_direct_property(vm.names.MAX_VALUE, Value(NumericLimits<double>::max()), 0);
-    define_direct_property(vm.names.MIN_VALUE, Value(NumericLimits<double>::min()), 0);
```

Comment:
You can't remove this in the previous commit, breaks bisecting.

Reply:
eek, let's squash them, then (3 commits here would be super overkill)

Reply:
you can keep it as two, just rename to `min_normal` initially and then fix it in commit 2 :)